packages/infrastructure/providers.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
```

Comment:
I feel like this stuff should maybe go in the `main.tf` file instead.

Reply:
In the future when some of the stuff in main.tf has been split into moduels